workout-tracker/src/views/AboutUs/AboutUs.js

```diff
@@ -248,25 +248,25 @@ const About = () => {
               <img src={sean} alt="profile" className="profile-img" />
               <h1>Shaun Carmody</h1>
               <p className="job-title">Team Lead</p>
-              <p className="about">
+              {/* <p className="about">
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
                 eiusmod tempor incididunt ut labore et dolore magna aliqua.
-              </p>
+              </p> */}
               <a href="http" className="btn">
                 Contact
               </a>
               <ul className="social-media">
                 <li className="social-media">
-                  <a href="http"><i className="fab fa-facebook-square"></i></a>
+                  <a href="https://www.facebook.com"><i className="fab fa-facebook-square"></i></a>
                 </li>
                 <li className="social-media">
-                  <a href="http"><i className="fab fa-twitter-square"></i></a>
+                  <a href="https://www.twitter.com"><i className="fab fa-twitter-square"></i></a>
                 </li>
                 <li className="social-media">
-                  <a href="http"><i className="fab fa-github-square"></i></a>
+                  <a href="https://github.com/shaunmcarmody"><i className="fab fa-github-square"></i></a>
                 </li>
                 <li className="social-media">
-                  <a href="http"><i className="fab fa-linkedin"></i></a>
+                  <a href="https://www.linkedin.com/in/shaunmcarmody/"><i className="fab fa-linkedin"></i></a>
                 </li>               
               </ul>
             </div>
```

Comment:
This page looks incredible! I think one way to improve it would be to put DIY into action here. It appears a lot of code is being repeated and could be refactored with an array of user objects and mapping over them to create the cards.

The above is just a stretch suggestion and that said the code already written works great.